pointcloud/basic_octree.go

```diff
@@ -39,6 +46,7 @@ type basicOctreeNode struct {
 	children []*BasicOctree
 	point    *PointAndData
 	maxVal   int
+	label    string
```

Comment:
I feel like you had it like this originally so this is a little annoying, but I think this makes more sense on the Octree object rather than the node object

Reply:
@jeremyrhyde you had wanted the label in the node, is there any reason?